src/quick-demo/README.md

```diff
@@ -0,0 +1,3 @@
+# Rainway "quick demo"
```

Comment:
nit: `src/<demo_name>` is inconsistent with our other example repos - does it make sense to treat this as two distinct examples (e.g. each with their own `package.json`) here?